src/server/handlers/launchpad.js

```diff
@@ -400,7 +402,8 @@ export const findSnaps = async (req, res) => {
       payload: {
         code: 'snaps-found',
         snaps: snaps
-      }
+      },
+      ...normalize(snaps, snapList)
```

Comment:
In this case we're not touching the payload, snaps refactor is for another pr, but we need snaps as entities for repositories to select on enabled repos.